src/lambda-secretLayer/wrapper16.js

```diff
@@ -7,9 +7,8 @@ async function getSecret() {
   
   AWS.config.update({region:process.env.AWS_REGION});
   const secretsManager = new AWS.SecretsManager();
-  //const secretName = process.env.SECRET_NAME;
-  const secretName = "lambda/coralogix/" + process.env.AWS_REGION + "/" + process.env.AWS_LAMBDA_FUNCTION_NAME;
-  console.log(secretName);
+  // const secretName = process.env.SECRET_NAME;
+  const secretName = process.env.CUSTOME_SECRET_NAME || "lambda/coralogix/" + process.env.AWS_REGION + "/" + process.env.AWS_LAMBDA_FUNCTION_NAME;  console.log(secretName);
```

Comment:
should it be custom ?

Reply:
Instead of using our secret, he is using his own custom secret that he will create. i thought that if i will just write a secret name it might cause confusion and the customer could think that this is the name of our secret.